src/validationinterface.h

```diff
@@ -34,6 +36,8 @@ class CValidationInterface {
     virtual void Inventory(const uint256 &hash) {}
     virtual void ResendWalletTransactions(int64_t nBestBlockTime) {}
     virtual void BlockChecked(const CBlock&, const CValidationState&) {}
+    virtual void GetScriptForMining(boost::shared_ptr<CReserveScript>&) {};
+    virtual void UpdateRequestCount(const CBlock&) {};
```

Comment:
Note that `Inventory(const uint256 &hash)`, connected to `g_signals.Inventory` does exactly the same thing. As far as I see we can get rid of `UpdateRequestCount` and replace it with that.